.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 matrix:
   include:
-    - python: 2.7
+    - python: 2.7.14
+      services: xvfb
```

Comment:
What does this mean?

Reply:
Xenial requires services to be manually started.  `xvfb` is required to run unit tests that require a plot.